src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -984,5 +984,8 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
     synonyms: {
       synonymsAPIDocumentation: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/current/synonyms-apis.html`,
     },
+    queryRules: {
+      queryRulesAPIDocumentation: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/current/query-rules-apis.html`,
```

Comment:
We should be linking to the new API reference from now on: https://www.elastic.co/docs/api/doc/elasticsearch/v9/operation/operation-query-rules-get-rule

@lcawl can you remind me of the latest guidance for linking to the the new API docs

Reply:
(as of 9.0 we'll be redirecting here from the old elasticsearch api page)